core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -705,4 +706,46 @@ default EventExecutor defaultSubscriberExecutor() {
         requireNonNull(options, "options");
         return StreamMessages.writeTo(map(mapper), destination, options);
     }
+
+    /**
+     * Adapts this {@link StreamMessage} to {@link InputStream}.
+     *
+     * <p>For example:<pre>{@code
+     * StreamMessage<String> streamMessage = StreamMessage.of("foo", "bar", "baz");
+     * InputStream inputStream = streamMessage.toInputStream(x -> HttpData.wrap(x.getBytes()));
+     * byte[] expected = ImmutableList.of("foo", "bar", "baz")
+     *                                .stream()
+     *                                .map(String::getBytes)
+     *                                .reduce(Bytes::concat).get();
```

Comment:
This example uses Guava's API. Could we simply replace this with `"foobarbaz".getBytes()`?

Reply:
✅ updated.